charts/artifacts/templates/deployment.yaml

```
@@ -20,6 +20,9 @@ spec:
         chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
         release: "{{ .Release.Name }}"
     spec:
+      {{- if .Values.deployment.terminationGracePeriodSeconds }}
+      terminationGracePeriodSeconds: {{ .Values.deployment.terminationGracePeriodSeconds }}
+      {{- end }}
```

I would set this value directly and apply a default value on the chart.
```suggestion
      terminationGracePeriodSeconds: {{ .Values.deployment.terminationGracePeriodSeconds }}
```